modules/serverless-inmemorychannel.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+//  * serverless/knative_eventing/serverless-channels.adoc
+
+[id="serverless-inmemorychannel_{context}"]
+= Using the default InMemoryChannel configuration
+
+InMemoryChannels are for development use only, and should not be used in a production environment.
+
+The following are limitations of InMemoryChannel channels:
+
+* No event persistence is available. If a Pod goes down, events on that Pod will be lost.
+* InMemoryChannel channels do not implement event ordering, so two events that are received in the channel at the same time may be delivered to a subscriber in any order.
+* If a subscriber rejects an event, there are no redelivery attempts. Instead, the rejected event is sent to a _dead letter sink_ if this sink exists, or is otherwise dropped.
+
+When you install Knative Eventing, the following custom resource definition is created automatically:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: knative-eventing
+  name: config-br-default-channel
+data:
+  channelTemplateSpec: |
+    apiVersion: messaging.knative.dev/v1
```

Comment:
looks like we use `messaging.knative.dev/v1` here but `messaging.knative.dev/v1beta1` below.